### DISCUSSION TITLE
Redis Cluster

### DISCUSSION THREAD

**Discussion by mgomse** [pimcore/pimcore#12535] [General]
In the [documentation](https://pimcore.com/docs/pimcore/current/Development_Documentation/Development_Tools_and_Details/Cache/index.html#page_Recommended-Redis-Configuration) there is written:

> Please note that the Redis adapter currently doesn't properly support Redis Cluster setups.

I've found that @dpfaffenbauer already asked about it [in issue from 2020](https://github.com/pimcore/pimcore/issues/5939) and issue was closed. 

My question is: **are there any plans to support [Redis Cluster](https://redis.io/docs/manual/scaling/)?**

(as I understand [Redis Sentinel](https://redis.io/docs/manual/sentinel/) architecture is already supported)

**Comment by dpfaffenbauer** [comment]
I think, but I am not 100% sure, that Pimcore X with Symfony Cache Integration supports a redis cluster.
